app/connectors/DownloadDataConnector.scala

```diff
@@ -53,14 +53,14 @@ class DownloadDataConnector @Inject() (config: FrontendAppConfig, httpClient: Ht
         }
       }
 
-  def getDownloadDataSummary(eori: String)(implicit hc: HeaderCarrier): Future[Option[DownloadDataSummary]] =
+  def getDownloadDataSummary(eori: String)(implicit hc: HeaderCarrier): Future[Option[Seq[DownloadDataSummary]]] =
     if (config.downloadFileEnabled) {
       httpClient
         .get(downloadDataSummaryUrl(eori))
         .execute[HttpResponse]
         .map { response =>
           response.status match {
-            case OK => Some(response.json.as[DownloadDataSummary])
+            case OK => Some(response.json.as[Seq[DownloadDataSummary]])
             case _  => None
```

Comment:
would there be a legitimate case where this would be none? if there are no summaries it just sends back an empty list so i don't think you need an option here